src/root/views/Country/ThreeW/NSProjects/index.tsx

```diff
@@ -145,18 +184,32 @@ function NSProjects(props: Props) {
           </div>
           <Container
             className={styles.ongoingProject}
-            heading="Ongoing projects"
-            actions="View All Projects >"
+            heading={viewAllProjects ? 'All Projects' : 'Ongoing Projects'}
+            actions={(
+              <Button
+                actions={<IoChevronForward />}
+                variant="tertiary"
+                onClick={toggleViewAllProject}
+              >
+                { viewAllProjects ? 'View Ongoing Projects' : 'View All Projects' }
```

Comment:
Add fixme to use string or use strings

Reply:
We'll be adding strings for FieldReport and 3W in bulk